examples/siro_sandbox/sandbox_app.py

```diff
@@ -859,7 +859,7 @@ def parse_debug_third_person(args, framebuffer_size):
     )
     parser.add_argument(
         "--can-grasp-place-threshold",
-        default=1.0,
+        default=1.2,
```

Comment:
I noticed an episode where I was unable to get close enough to the fridge to place my object. Ruta can test and tune this further in the future.

Reply:
Yes, this happened to me too. 

Also, if the can grasp radius is large there may be the cases when the objects can be picked up through the walls or obstacles. Unfortunately, I didn't come up with the idea how that could be addressed.